guides/common/modules/proc_performing-discovery-in-pxeless-mode.adoc

```diff
@@ -0,0 +1,57 @@
+[id="performing-discovery-in-pxeless-mode_{context}"]
+= Performing Discovery in PXE-less mode
+[[Implementing_PXE_less_Discovery_{context}]]
+
+Discovery in PXE-less mode uses the Discovery ISO and requires you to attend to the process.
+
+.Prerequisites
+ifdef::satellite[]
+* You have installed the Foreman Discovery image.
+For more information, see xref:Installing_the_Discovery_Service_{context}[].
+endif::[]
+ifndef::satellite[]
+* You have installed the Foreman Discovery image or built a custom Discovery image ISO.
+For more information, see one of the following resources:
+** xref:Installing_the_Discovery_Service_{context}[]
+** xref:Building_a_Discovery_Image_{context}[]
+endif::[]
+
+.Procedure
+. Copy the Discovery ISO to either a CD, DVD, or a USB stick.
+For example, to copy to a USB stick at `/dev/sdb`:
++
+[options="nowrap" subs="+quotes"]
+----
+# dd bs=4M \
+if=/usr/share/foreman-discovery-image/foreman-discovery-image-_version_.iso \
+of=/dev/sdb
+----
+. Insert the Discovery boot media into a bare-metal host, start the host, and boot from the media.
+The Discovery Image displays an option for either *Manual network setup* or *Discovery with DHCP*:
++
+
+* If you select *Manual network setup*, the Discovery image requests a set of network options.
```

Comment:
```suggestion
* *Manual network setup*: Enter the following options:

** Primary network interface
** IPv4 Address
** etc.
```

Reply:
It would be a good idea to find out what is on the screen. 

Otherwise, we're basically telling the user, "If you select 'x', fill in the options. If you select 'y', fill in different options." :-) 

If the options themselves are not important, you could change the lead-in sentence to a step: "Select **Manual network setup** or **Discovery with DHCP**, enter the required fields, and click **Next**."